AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/edit-work-hours.component.ts

```diff
@@ -99,13 +101,24 @@ export class EditWorkHoursComponent implements OnInit {
         this.result = $event;
         this.showWorkHoursTable = false;
         this.showNonWorkHoursTable = false;
-        if (this.result) {
-            this.showWorkHoursTable = this.result[0] instanceof VhoWorkHoursResponse;
-            this.showNonWorkHoursTable = this.result[0] instanceof VhoNonAvailabilityWorkHoursResponse;
+        switch (this.hoursType) {
+            case HoursType.WorkingHours:
+                this.showWorkHoursTable = true;
+                this.showNonWorkHoursTable = false;
```

Comment:
not sure this is needed, it's already been set to false above

Reply:
just to be on the safe side if it will be called after an external event

Reply:
sorry not sure what you mean by that? It's already being set to false in line 103, and that's the only entry point to the function